Sources/HTTP/Models/Headers/Headers.swift

```diff
@@ -42,6 +42,245 @@ extension HeaderKey {
     }
 }
 
+extension HeaderKey {
+
+  //
+  // Common HTTP Header Request keys
+  //
+
+  static public var Accept: HeaderKey {
```

Comment:
Swift guidelines state member properties should be camelCase with first letter lowercase.